includes/class-freemius.php

```diff
@@ -15832,9 +15833,8 @@ function get_site_info( $site = null, $load_registration = false ) {
                 $info[ 'registration_date' ] = $registration_date;
             }
 
-            if ( $switched ) {

```

Comment:
@fajardoleo why are we removing this? Don't we want to restore the blog like before?